Atomic/atomic.py

```diff
@@ -1050,6 +1078,17 @@ def set_debug(self):
         if self.args.debug:
             self.debug = True
 
+    def get_all_vulnerable_info(self):
+        """
+        Will simply read and return the entire /var/lib/atomic/scan_summary.json as a JSON string so it can then be parsed appropriately.
+        """
+        try:
+            return json.dumps(json.loads(open(os.path.join(self.results, "scan_summary.json"), "r").read()))
+            #return json.dumps(summary_results)
```

Comment:
i removed the commented code. also yeah when i had tried it in the past it didn't seem to work but i gave another attempt and now it looks good. i'll implement those changes.